app/javascript/mastodon/features/ui/components/onboarding_modal.js

```diff
@@ -191,28 +214,36 @@ class OnboardingModal extends React.PureComponent {
     this.setState({ currentIndex: i });
   }
 
+  handlePrev = () => {
+    this.setState(({ currentIndex }) => ({
+      currentIndex: Math.max(0, currentIndex - 1),
+    }));
+  }
+
   handleNext = () => {
+    const { pages } = this.state;
```

Comment:
What is the reason for storing pages in state? It doesn't even need to be props, I think it's a constant.